File: lib/go/rpc/ResponseMsg.go
package rpc

const ResponseMsg_strFingerprint = "d44e3caf8903d1f381a22194a181c1cc"

type ResponseMsg struct {
	Header   RpcHeader `json:"header"`
	MsgSeqId int64     `json:"msgSeqId"`
	MsgId    int32     `json:"msgId"`
	Buff     []byte    `json:"buff"`
}

func (this *ResponseMsg) GetFingerprint() string {
	return ResponseMsg_strFingerprint
}
func (this *ResponseMsg) Serialize(P__ IProtocol) {
	this.Header.Serialize(P__)

	P__.WriteInt64(this.MsgSeqId)

	P__.WriteInt32(this.MsgId)

	P__.WriteBinary(this.Buff)

}
func (this *ResponseMsg) DeSerialize(P__ IProtocol) error {
	this.Header.DeSerialize(P__)

	this.MsgSeqId = P__.ReadInt64()

	this.MsgId = P__.ReadInt32()

	this.Buff = P__.ReadBinary()

	return nil
}

File: lib/ts/rpc/ResponseMsg.ts
import  * as rpc from "../rpc/index"
import  {RpcHeader}  from "./RpcHeader"
export class ResponseMsg
{ 
	public static readonly strFingerprint:string="d44e3caf8903d1f381a22194a181c1cc"
	public header:RpcHeader=new RpcHeader()
	public msgSeqId:rpc.Int64=new rpc.Int64
	public msgId:number=0
	public buff:Uint8Array|null=null
	//serialize
	public serialize( __P__:rpc.IProtocol):void 
	{ 
		this.header.serialize(__P__);

		__P__.writeInt64(this.msgSeqId);

		__P__.writeInt32(this.msgId);

		__P__.writeBinary(this.buff);

	}// serialize
	//deSerialize
	public deSerialize( __P__:rpc.IProtocol):void 
	{ 
		this.header.deSerialize(__P__)

		this.msgSeqId=__P__.readInt64()

		this.msgId=__P__.readInt32()

		this.buff=__P__.readBinary()

	}// deSerialize
}//class
